src/components/non-inferiority-comp.vue

```diff
@@ -1,213 +1,188 @@
-            },
-            set (newValue) {
-                this.$store.dispatch('field:change', {
-                    prop: 'expectedChange',
-                    value: newValue
-                })
-            }
-        },
+    enabled() {
+      return this.$store.getters.isNonInferiority
     },
-    methods: {
-        enableInput () {
-            this.$emit('edit:update', {prop: 'base'})
-        },
-        updateFocus ({fieldProp, value}) {
-            if (this.focusedBlock == fieldProp && value === false) {
-                this.focusedBlock = ''
-            } else if (value === true) {
-                this.focusedBlock = fieldProp
-            }
-
-            this.$emit('update:focus', {
-                fieldProp: this.fieldFromBlock,
-                value: value
+    thresholdRelative: {
+      get() {
+        return this.$store.getters.relativeThreshold
+      },
+      set(threshold) {
+        if (this.focusedBlock === FOCUS.SAMPLE) {
+          if (this.relativeDebouncer != null) {
+            clearTimeout(this.relativeDebouncer)
+          }
+          this.relativeDebouncer = setTimeout(() => {
+            this.$emit('update:selected', SELECTED.RELATIVE)
+            this.$store.commit('SET_THRESHOLD', {
+              threshold,
+              isAbsolute: false,
+              lockedField: this.lockedField,
             })
+          }, DEBOUNCE)
         }
-
-    }
+      },
+    },
+    thresholdAbsolute: {
+      get() {
+        return this.$store.getters.absoluteThreshold
+      },
+      set(threshold) {
+        if (this.focusedBlock === FOCUS.SAMPLE) {
+          if (this.absoluteDebouncer != null) {
+            clearTimeout(this.absoluteDebouncer)
+          }
+          this.absoluteDebouncer = setTimeout(() => {
+            this.$emit('update:selected', SELECTED.ABSOLUTE)
+            this.$store.commit('SET_THRESHOLD', {
+              threshold,
+              isAbsolute: true,
+              lockedField: this.lockedField,
+            })
+          }, DEBOUNCE)
+        }
+      },
+    },
+    onlyTotalVisitors() {
+      return this.$store.getters.trafficMode === TRAFFIC_MODE.TOTAL
+    },
+    testType() {
+      return this.$store.getters.testType
+    },
+  },
 }
 </script>
 
 <style>
 .pc-non-inf-select {
-    --base-padding: 5px;
-    font-size: inherit;
-    line-height: 28px;
-    border: none;
-    display: block;
-    position: relative;
-    box-sizing: border-box;
-    width: 100%;
-    filter: drop-shadow(0 4px 2px rgba(0,0,0,0.1));
-    border-radius: 5px;
-    background: var(--white);
-    padding: var(--base-padding);
-
-    overflow: hidden;
-    -webkit-appearance: none;
-    -moz-appearance: none;
-    appearance: none;
+  --base-padding: 5px;
+  font-size: inherit;
+  line-height: 28px;
+  border: none;
+  display: block;
+  position: relative;
+  box-sizing: border-box;
+  width: 100%;
+  filter: drop-shadow(0 4px 2px rgba(0, 0, 0, 0.1));
```

Comment:
Why not using box-shadow?

Reply:
No idea. To be honest, I have not touched at all the original CSS.